### DISCUSSION TITLE
trivy connects to `{s3b,snd}.wx-one.com` resources

### DISCUSSION THREAD

**Discussion by aschaber1** [aquasecurity/trivy-operator#1895] [Development]
Hello, 

I'm not a 100% sure, but I believe `trivy-operator` is connecting to
- `https://s3b.wx-one.com/`
- `https://snd.wx-one.com/`

It's causing quiet a bit of traffic, hence I am curious why it connecting and if I can mirror it locally.

Any advice is appreciated.

Best regards
Alexander

**Comment by wzrdtales** [comment]
https://wx1.de CEO here. Someone is most likely consuming content from https://opencode.de in your Organization. Should they consume the docker registry, or package registry, there is a chance you can set up a local mirror for it.

**Reply by chen-keinan** [reply]
@aschaber1 @wzrdtales  There is no reference in the `Trivy-operator` code to any of the mentioned resources. All our resources are hosted on Aqua's GHCR.

Perhaps one of the community users configured or deployed an image that references these resources and used `trivy-operator` to scan it.

**Note:** Trivy-Operator is an open-source project with a large community of users

**Reply by aschaber1** [reply]
> https://wx1.de CEO here. Someone is most likely consuming content from https://opencode.de in your Organization. Should they consume the docker registry, or package registry, there is a chance you can set up a local mirror for it. 

Hey, thank you for helping out. Makes sense now ☺️ we indeed had some images from opencode.de

Best regards
Alexander
